documentation/DesignDocumentation.md

```diff
@@ -6,24 +6,48 @@ The two main forms of design that are most relevant to the Tornelo Scoresheet ap
 
 **System Architecture**
 
-The System Architecture includes the way the model, the way the program is structured and the programming languages and tools used. 
+The System Architecture includes the way the model, the way the program is structured and the programming languages and tools used.
 
-Key areas of this aspect of the system design and rationale behind relevant design choices are documented in Section 1 of this document. 
+Key areas of this aspect of the system design and rationale behind relevant design choices are documented in Section 1 of this document.
 
 **User Interface (UI) Design**
 
-The User Interface Design is a holistic approach to deciding how the applications functional elements are displayed on the screen. This includes considering the users and their experiences, as well as the necessary features and functional requirements. 
+The User Interface Design is a holistic approach to deciding how the applications functional elements are displayed on the screen. This includes considering the users and their experiences, as well as the necessary features and functional requirements.
 
-Key areas of this aspect of the system design and rationale behind relevant design choices are documented in Section 1 of this document. 
+Key areas of this aspect of the system design and rationale behind relevant design choices are documented in Section 1 of this document.
 
 ## 1 System Architecture
-TODO
 
-### 1.1 Class Diagram and Model
-TODO
-
-### 1.2 Folder Structure
-TODO
+### 1.1 Architecture/Model Diagram
+![architecture_diagram](Pictures/system_design-dependency_structure.png)
+
+The project's architecture is broken down to 6 parts:
+1. Pages
+    * Serves a distinct user function, as described in section 2 - Wireframes
+    * Contains components
+2. Components
+    * Parts of a page
+    * Can contain other components
+3. Context
```

Comment:
We have changed the way we are handling state, so I think we should include here the fact that each page will use it's own custom hook, and these hooks each access the context, but each page does not load the context itself